endpointmanager/pkg/endpointmanager/postgresql/fhirendpointmetadatastore_test.go

```diff
@@ -254,5 +256,77 @@ func Test_PersistFHIREndpointMetadata(t *testing.T) {
 	if count != 1 {
 		t.Errorf("expected 1 update for endpointInfo1 in history table. Got %d.", count)
 	}
+}
+
+func Test_AvailabilityUsesMetadataRequestedVersion(t *testing.T) {
```

Comment:
Should a test case potentially be added that ensures that adding in another metadata entry with the same URL and requested version does in fact update the availability correctly? As of right now all the requested versions are different and so there is not really an explicit test that checks to make sure the availability gets updated for the same URL and requested version